exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SQLExpressionBuilder.kt

```diff
@@ -220,14 +220,14 @@ fun Sequence.nextLongVal(): NextVal<Long> = NextVal.LongNextVal(this)
 // Value Expressions
 
 /** Specifies a conversion from one data type to another. */
-fun <R> Expression<*>.castTo(columnType: IColumnType): ExpressionWithColumnType<R> = Cast(this, columnType)
+fun <R> Expression<*>.castTo(columnType: IColumnType<R & Any>): ExpressionWithColumnType<R> = Cast(this, columnType)
 
 // Misc.
 
 /**
  * Calls a custom SQL function with the specified [functionName] and passes this expression as its only argument.
  */
-fun <T : Any?> ExpressionWithColumnType<T>.function(functionName: String): CustomFunction<T?> = CustomFunction(functionName, columnType, this)
+fun <T> ExpressionWithColumnType<T>.function(functionName: String): CustomFunction<T?> = CustomFunction(functionName, columnType, this)
```

Comment:
[The default upper bound (if there was none specified) is `Any?`.](https://kotlinlang.org/docs/generics.html#generic-constraints:~:text=The%20default%20upper%20bound%20(if%20there%20was%20none%20specified)%20is%20Any%3F)